azure-pipelines.yml

```diff
@@ -0,0 +1,115 @@
+# Mutilated beyond recognition from the example at:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/go
+
+trigger:
+- v2
+
+schedules:
+- cron: "0 0 * * *"
```

Comment:
What's with the schedule, again?

Reply:
I left there just in case a direct commit, i.e. not via PR, go into the project.